backend/typescript/graphql/types/taskType.ts

```diff
@@ -0,0 +1,63 @@
+import { gql } from "apollo-server-express";
```

Comment:
TODO: make sure the types on this file (taskType.ts), interfaces, and schema.prisma all match with spelling/casing/types etc